src/Administration/Resources/app/administration/src/app/component/form/select/entity/sw-entity-single-select/sw-entity-single-select.html.twig

```diff
@@ -80,9 +80,13 @@
                                     </template>
                                 {% endblock %}
 
-                                {% block sw_entity_single_select_base_results_list_after %}
+                                {% block sw_entity_single_select_base_results_list_pagination %}
```

Comment:
renaming a block is a break. could you please add a new block with the new name and deprecate the old one?